ts/components/core/typography/H2.tsx

```diff
@@ -16,6 +16,7 @@ type OwnProps = ExternalTypographyProps<
 
 const fontName: IOFontFamily = "TitilliumWeb";
 export const h2FontSize = 20;
+export const h2LineHeight = h2FontSize * 1.2;
```

Comment:
as I see in [H1 component](https://github.com/pagopa/io-app/pull/4093/files#diff-1ebe46ed252e9930166384fd3c141b6f21cdbb6bed6a0cb5dd350a1dc6d43a5dR16) wouldn't it be better to keep the absolute value here instead of calculating the lineHeight?

Reply:
There's no right or wrong approach here. In the first iteration, I preferred to unlink the font size value from the line height one, so you don't have to update both values every time one of the values changes. The instruction above also scales pretty well if you have a range of various line-height values (from strict `1.1` to loose `1.5`) that you can adopt based on your needs.

For the moment, let's keep the absolute value so we have a consistent code between the typographic styles.